packages/sdks/overrides/react-native/src/functions/transform-style-property.ts

```diff
@@ -19,11 +19,11 @@ function extractVarValue(value: string): string {
   return newValue;
 }
 // Common regex patterns
-const numberPattern = /^-?\d+(\.\d+)?$/;
-const lengthPattern = /^-?(\d+)(px|%)?$/;
-const pixelPattern = /^-?(\d+)(px)?$/;
+const numberPattern = /^-?\d*\.?\d+$/;
+const lengthPattern = /^-?\d*\.?\d+(px|%)?$/;
+const pixelPattern = /^-?\d*\.?\d+(px)?$/;
```

Comment:
is the idea to allow floats? what was wrong with the previous regex?

Reply:
yeah floats